### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Mohanraj209**

### CODE REVIEW

File: deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv [hunk 1]
```diff
@@ -260,21 +228,1172 @@ registration-processor-default.properties,GETVIDSBYUIN
...
+registration-processor-default.properties,GETUINBYVID
+registration-processor-default.properties,CREDENTIALREQUEST
+registration-processor-default.properties,GETVIDSBYUIN
+registration-processor-default.properties,ENCRYPTURL
+registration-processor-default.properties,ENCRYPTIONSERVICE
+registration-processor-default.properties,DIGITALSIGNATURE
+registration-processor-default.properties,CRYPTOMANAGERDECRYPT
+registration-processor-default.properties,SMSNOTIFIER
+registration-processor-default.properties,EMAILNOTIFIER
+registration-processor-default.properties,PMS
```
Comment (vishwa-vyom) [inline deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv:1390]: there is something wrong here, it show too many new lines added, but we are only deleting entries from all 3 files.
In this file we are suppose to delete 42 entries, that should be the only change

File: deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv [hunk 2]
```diff
@@ -46,88 +46,14 @@ registration-processor-default.properties,packetmanager.provider.uingenerator.pa
...
-registration-processor-default.properties,registration.processor.reprocess.minutes:
-registration-processor-default.properties,mosip.kernel.virus-scanner.port:
-registration-processor-default.properties,registration.processor.queue.username:
-registration-default.properties,mosip.registration.num_of_fingerprint_retries
-registration-default.properties,mosip.registration.num_of_iris_retries
-registration-default.properties,mosip.registration.num_of_face_retries
-registration-default.properties,mosip.registration.leftslap_fingerprint_threshold
-registration-default.properties,mosip.registration.rightslap_fingerprint_threshold
-registration-default.properties,mosip.registration.thumbs_fingerprint_threshold
-registration-default.properties,mosip.registration.iris_threshold
```
Comment (vishwa-vyom) [inline deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv:62]: this have only 5 entries removed

File: deployment/v3/utils/prop_migrator/knowledge/new-property-with-decent-default-value.csv [hunk 3]
```diff
@@ -1406,14 +1274,13 @@ resident-default.properties,sequence-order
...
 resident-default.properties,resident.template.ack.update-demographic-data
-resident-default.properties,resident.update-uin.machine-name-prefix 
+resident-default.properties,resident.update-uin.machine-name-prefix
 resident-default.properties,mosip.scope.resident.postAuthTypeStatus
 resident-default.properties,resident.template.tnc.share-cred-with-partner
 resident-default.properties,mosip.resident.ack.personalised_card.name.convention
 resident-default.properties,resident.template.bell-icon.success.update-demo-data
 resident-default.properties,resident.template.bell-icon.request-received.verify-my-phone-email
 resident-default.properties,resident.template.email.subject.success.vid-card-download
-resident-default.properties,otpChannel.mobile
```
Comment (vishwa-vyom) [inline deployment/v3/utils/prop_migrator/knowledge/new-property-with-decent-default-value.csv:1416]: this should have only 133 lines removed

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: deployment/v3/utils/prop_migrator/knowledge/new-property-with-decent-default-value.csv (hunk 3 above)
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/knowledge/new-property-with-decent-default-value.csv:1416]: I have checked the count, the number of properties present now is 1625 and previously we had 1758. so 1758-1625=133 the properties removed is accurate sir.

File: deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv (hunk 2 above)
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv:62]: apart from yes and yes-vish we also removed the #N/A properties aswell sir that's why the count is high.

File: deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv (hunk 1 above)
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv]: updated sir.

File: deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv [hunk 4]
```diff
@@ -325,82 +284,6 @@ resident-default.properties,CREDENTIAL_TYPES_URL
...
-application-default.properties,mosip.kernel.rid.timestamp-length
-application-default.properties,mosip.kernel.rid.sequence-length
-application-default.properties,mosip.kernel.otp.expiry-time
-application-default.properties,mosip.primary-language
-application-default.properties,mosip.secondary-language
-application-default.properties,mosip.kernel.applicant.type.age.limit
-application-default.properties,mosip.idrepo.identity.allowedBioAttributes
-application-default.properties,mosip.idrepo.identity.bioAttributes
+registration-processor-default.properties,registration.processor.reprocess.minutes
+registration-processor-default.properties,registration.processor.queue.username
```
Comment (vishwa-vyom) [inline deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv:288]: As per the comment and delete marking these both to be deleted

File: deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv (hunk 2 above)
Comment (vishwa-vyom) [reply deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv:62]: in old value list, NA has only 2 records, also we din't discuss to remove NA for old value takes priority

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv (hunk 2 above)
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv:62]: as per the discussion we have removed the duplicates which are present in both latest as well as old takes priority.

File: deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv (hunk 4 above)
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv]: updated sir.

File: deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv (hunk 2 above)
Comment (vishwa-vyom) [reply deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv:62]: No we have not discussed that, we have to remove NA only in the lastest takes priority, items in the old one should be retained. Also the count is not matching.
Items marked for delete using yes, yes-vish is only 5 items.
Items marked as NA is only 2 items in the old takes priority list.
But lines deleted is much more than the combined count which is 7

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**